src/app/components/core/control-dispatcher/api-flow/api-flow.component.html

```diff
@@ -0,0 +1,211 @@
+                            <span class="is-small"><!--{{ep.type | titlecase}}-->Operation</span>
+                            <span class="help sub">
+                                <!--{{ep.type == 'operation' ? ep.operation_name : ''}} -->
+                                Off
+                            </span>
+                        </td>
+                        <td></td>
+                        <td class="align-content">
+                            <span class="is-small"><!--{{ep.destination}} --> Any</span>
+                            <span class="help sub">
+                                <!-- {{ ['any', 'broadcast'].includes(ep.destination?.toLowerCase()) ? '*' : ep.destination_name }} -->
+                                *
+                            </span>
+                        </td>
+
+                        <td class="align-content">
+                            <ng-container *ngIf="rolesService.hasControlAccess(); else staticText">
+                              <input class="checkbox" type="checkbox" [checked]="ep?.permitted"
+                                (click)="onCheckboxClicked($event, ep.name)">
+                            </ng-container>
+                            <ng-template #staticText>
+                              <input disabled class="checkbox" type="checkbox" [checked]="ep?.permitted">
+                            </ng-template>
+                        </td>
+
+                        <td>
+                        <!-- TODO: Show to Editor or above role based users only, if auth required -->
+                        <div *ngIf="rolesService.hasControlAccess()" class="dropdown is-left is-hoverable">
+                            <div class="dropdown-trigger">
+                            <a class="button is-small context-menu" aria-haspopup="true" aria-controls="dropdown-menu">
+                                <span class="icon">
+                                <i class="fas fa-ellipsis-v fa-sm fa-fw text-gray-600"></i>
+                                </span>
+                            </a>
+                            </div>
+                            <!-- if view role OR  request is not allowed, then don't show empty drop menu-->
+                            <div class="dropdown-menu" id="dropdown-menu" role="menu">
+                            <div class="dropdown-content">
+                                <!-- TODO: FOGL-8037
+                                Add check of logged-in user vs allowed users (Editor or above role) if not anonymous 
+                                Show modal to override variable values
+
+                                There is no enough info on list page. So, only checked for permitted
+                                -->
+                            
+                            <a class="dropdown-item" *ngIf="ep.permitted" (click)="getAPIFlow(ep.name);openModal('confirmation-execute-dialog', ep.name, ep.description)">
```

Comment:
This check is WRONG!

We must check for each record the same way, we have in details page for this.allowExecute, i.e. either anonymous or loggedInUser? exists in allow list. Add todo what if an Admin or, below the role of Editor.